code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -48,6 +50,18 @@
 	sawn_desc = "Come with me if you want to live."
 	can_be_sawn_off  = TRUE
 
+/obj/item/gun/ballistic/shotgun/riot/sawoff(mob/user)
+	. = ..()
+	if(.)
+		weapon_weight = WEAPON_MEDIUM
```

Comment:
this is a balance change.

Reply:
Reverted, was pulled from the other shotgun sawoff.